build.gradle

```diff
@@ -0,0 +1,66 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.3.0'
+    classpath 'com.f2prateek.javafmt:javafmt:0.1.4'
+    classpath 'com.vanniktech:gradle-android-junit-jacoco-plugin:0.6.0'
+  }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'checkstyle'
+apply plugin: 'com.f2prateek.javafmt'
+apply plugin: 'com.vanniktech.android.junit.jacoco'
+
+android {
+  compileSdkVersion 26
+  buildToolsVersion '25.0.3'
+  defaultConfig {
+    minSdkVersion 14
+    targetSdkVersion 26
+  }
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_7
+    targetCompatibility JavaVersion.VERSION_1_7
+  }
+  testOptions {
+    unitTests {
+      all {
+        jvmArgs '-noverify'
+      }
+    }
+  }
+  lintOptions {
+    textReport true
+    textOutput 'stdout'
+    abortOnError false
+  }
+  productFlavors {}
+}
+
+dependencies {
+  repositories {
+    mavenCentral()
+  }
+  compile 'com.segment.analytics.android:analytics:4.0.0'
```

Comment:
It looks like the latest stable release is 4.2.6. Are we using that for new integrations, or 4.0.0?

Reply:
It doesn't really matter on Android.